projects/example-app/src/app/auth/components/logout-confirmation-dialog.component.spec.ts

```diff
@@ -1,13 +1,13 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { MatButtonModule, MatDialogModule } from '@angular/material';
```

Comment:
Should we just be more specific instead?
```ts
import { MatButtonModule } from '@angular/material/button';
import { MatDialogModule } from '@angular/material/dialog';
```

Reply:
> BREAKING CHANGE: Components can no longer be imported through "@angular/material". Use the individual secondary entry-points.

from https://github.com/angular/components/commit/0c1d369dcfca506f4017f36e0a7a0331855dcae3